filters/ThirdParty/filter_236_ROad-Block/template.txt

```diff
@@ -1,2 +1,3 @@
 !
-@include "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-block-filters-light.txt" /exclude="../../exclusions.txt"
\ No newline at end of file
+@include "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-block-filters-light.txt" /exclude="../../exclusions.txt"
+@include "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-ubo.txt" /exclude="../../exclusions.txt"
```

Comment:
There is:
```adblock
!#include road-ubo.txt
```
at the bottom of this file - https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-block-filters-light.txt
So, will it not be automatically included?

Reply:
Checked this locally and you were right. Removed the url.